### PR TITLE
Move Chart.js to Composer

### PR DESCRIPTION
**Pull request by MrPetovan**
Part of #4265

This is a simple move as Chart.js doesn't have any dependencies.

### CODE REVIEW

File: view/templates/admin/federation.tpl [hunk 1]
```diff
@@ -36,15 +63,42 @@ var myDoughnutChart = new Chart(ctx).Doughnut(FedData, { animateRotate : false,
...
+		}],
+		labels: [
+	{{foreach $c[1] as $v}}
+			'{{$v['version']}}',
+	{{/foreach}}
+		]
+	};
     var ctx = document.getElementById("{{$c[2]}}Chart").getContext("2d");
-    var my{{$c[2]}}DoughnutChart = new Chart(ctx).Doughnut({{$c[2]}}data, {animateRotate : false,});
+	var my{{$c[2]}}DoughnutChart = new Chart(ctx, {
```
Comment (tobiasd) [inline view/templates/admin/federation.tpl]: the indentation looks wrong in githubs diff. can you check?

Comment (MrPetovan) [reply view/templates/admin/federation.tpl]: Indeed, I used tabs when it was spaces.

File: view/global.css [hunk 2]
```diff
@@ -399,14 +399,9 @@ a {
   color: #00a700;
 }
 .federation-graph {
-  width: 400px;
-  height: 400px;
-  float: right;
```
Comment (tobiasd) [inline view/global.css]: why was the float removed?

Comment (MrPetovan) [reply view/global.css]: Because it doesn't look good right now. 😛

Comment (tobiasd) [reply view/global.css]: You have redesigned the page? The floating was added to avoid to much dead space aside of the graph.

Comment (MrPetovan) [reply view/global.css]: My current version has dead space only on the left of the graph, so I centered it horizontally instead.

Comment (tobiasd) [reply view/global.css]: So now there is dead space left and right aside of the graph and aside of the project listing. Well if you like it that way better ;-)

I would prefer the heading of the page being the first thing on the page as well. The way it looks now, for me, is that the graph is totally unrelated to the rest of the page. Before the stats were aside of the graph so they were related.

Comment (MrPetovan) [reply view/global.css]: Ah, but it never looked this way on frio, hence the confusion between us.

Comment (tobiasd) [reply view/global.css]: Oh, that is bad. Anyhow the way it looks now is worse then before IMO I'll revert this merge maybe ;-)

Comment (MrPetovan) [reply view/global.css]: Please don't, this wasn't a display merge, but a library move. If you want to improve the display in vier, please add your changes to the theme.

**Comment by tobiasd** [comment]
> Please don't, this wasn't a display merge, but a library move. If you
> want to improve the display in vier, please add your changes to the
> theme.

Yes, of course. By reverting your changes, I meant the one for the template, not the library stuff.

**Pull request merged**